test/models/admin_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class AdminTest < ActiveSupport::TestCase
+  # test "the truth" do
```

Comment:
You don't have to do it in this PR, cuz it's already so big, but you do another PR to add tests. On this team, we pride ourselves in good test coverage. We are trying to move to Rspec so maybe see about a tutorial on that. See examples in the LM-backend repo. Also, generally we want to practice TDD, check out https://medium.com/@WesleyDavis/rspec-and-test-driven-development-618b44c8177f